src/Spd.Manager.Printing/PrintingManager.cs

```diff
@@ -64,10 +80,32 @@ private static DocumentTransformRequest CreateDocumentTransformRequest(PrintJob
         };
     }
 
-    private async Task<string> PrintViaBcMailPlus(BcMailPlusTransformResponse bcmailplusResponse, CancellationToken cancellationToken)
+    private async Task<string> PrintViaBcMailPlus(BcMailPlusTransformResponse bcmailplusResponse, Guid eventId, CancellationToken cancellationToken)

```

Comment:
The name of this method and what it is doing (update) is a bit misleading. Should we change the name?

Reply:
add more error processing and refactor the code.